src/sbvr-api/permissions.ts

```diff
@@ -1220,76 +1223,76 @@ export const checkPassword = async (
 	if (user == null) {
 		throw new Error('User not found');
 	}
-	const hash = user.password;
-	const userId = user.id;
-	const actorId = user.actor;
-	const res = await sbvrUtils.sbvrTypes.Hashed.compare(password, hash);
+	const res = await sbvrUtils.sbvrTypes.Hashed.compare(password, user.password);
 	if (!res) {
 		throw new Error('Passwords do not match');
 	}
+	const userId = user.id;
 	const permissions = await getUserPermissions(userId);
 	return {
 		id: userId,
-		actor: actorId,
```

Comment:
How was that working???

Reply:
Presumably nothing depended on the resultant actor id being the number it said it was